src/main/java/com/coveo/feign/ReflectionErrorDecoder.java

```diff
@@ -83,7 +82,9 @@ public ReflectionErrorDecoder(
 
       for (Method method : apiClass.getMethods()) {
         if (method.getAnnotation(RequestLine.class) != null
-            || (isSpringWebAvailable && method.getAnnotation(RequestMapping.class) != null)) {
+            || (isSpringWebAvailable && method.getAnnotation(RequestMapping.class) != null)
+            || Stream.of(method.getAnnotations())
```

Comment:
Your addition could replace the `(isSpringWebAvailable && method.getAnnotation(RequestMapping.class) != null)` condition if I'm not mistaken, it is redundant otherwise...

Also, your addition breaks the optional dependency on spring, if the plugin is used in a non spring environment, this will throw a `ClassNotFoundException`. It's missing the `isSpringWebAvailable` boolean.